packages/jellyfish-api-core/__tests__/category/loan/paybackLoan.test.ts

```diff
@@ -1535,7 +1690,7 @@ describe('paybackloan for dusd using dfi', () => {
       from: vaultOwnerAddress
     })
 
-    await expect(promise).rejects.toThrow('RpcApiError: \'Test PaybackLoanTx execution failed:\nPayback of DUSD loans with DFI not currently active\', code: -32600, method: paybackloan')
+    await expect(promise).rejects.toThrow('RpcApiError: \'Test PaybackLoanTx execution failed:\nPayback of loan via DFI token is not currently active\', code: -32600, method: paybackloan')
   })
 
   it('should not be able to payback TSLA loan using DFI', async () => {
```

Comment:
`should not be able to payback TSLA loan using DFI` is not true. If governance allows it, it should be able to do so.
But however, with the current ain side implementation, if you are using `amounts` property instead of `loans`, you can only pay DUSD with DFI. not TSLA with DFI
In a multi loan token vault, `defid` can not choose which one to repay since `amounts` do not specify it when paying with DFI
Ref - https://github.com/DeFiCh/ain/pull/1111/files#diff-5b5f5b126aad5172f5e15066f2926f4afb57e75eb624b509f56233c14cd4b5c9R2801-R2803

Reply:
maybe you can change the test description accordingly.

Reply:
Also add another test to pay `DUSD` using `TSLA` with `loans` property. and check for failure.

Reply:
one more test to check for failure when `loans` and `amounts` both are specified.